Mathlib/RingTheory/Localization/Basic.lean

```diff
@@ -134,6 +134,74 @@ protected lemma bijective (f : S →+* Q) (hf : f.comp (algebraMap R S) = algebr
 
 end AlgEquiv
 
+section liftAlgHom
+
+variable {A : Type*} [CommSemiring A]
+  {R : Type*} [CommSemiring R] [Algebra A R] {M : Submonoid R}
+  {S : Type*} [CommSemiring S] [Algebra A S] [Algebra R S] [IsScalarTower A R S]
+  {P : Type*} [CommSemiring P] [Algebra A P] [IsLocalization M S]
+  {f : R →ₐ[A] P} (hf : ∀ y : M, IsUnit (f y)) (x : S)
+include hf
+
+/-- `AlgHom` version of `IsLocalization.lift`. -/
+noncomputable def liftAlgHom : S →ₐ[A] P :=
```

Comment:
I think we usually call it `liftₐ`

Reply:
There are also several maps calling `xxxAlgHom` in mathlib, for example [Polynomial.mapAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/AlgebraMap.html#Polynomial.mapAlgHom), [Polynomial.CAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/AlgebraMap.html#Polynomial.CAlgHom), [MvPolynomial.mapAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/MvPolynomial/Basic.html#MvPolynomial.mapAlgHom), etc. As for `liftAlgHom`, there are [RingQuot.preLiftAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/RingQuot.html#RingQuot.preLiftAlgHom), [RatFunc.liftAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/RatFunc/Basic.html#RatFunc.liftAlgHom) and [PiTensorProduct.liftAlgHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/PiTensorProduct.html#PiTensorProduct.liftAlgHom).

The only `liftₐ` in mathlib is [Ideal.Quotient.liftₐ](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Quotient/Operations.html#Ideal.Quotient.lift%E2%82%90), but there are other `ₐ` such as [IsLocalization.mapₐ](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/Algebra.html#IsLocalization.map%E2%82%90) and [AdicCompletion.evalₐ](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/AdicCompletion/Algebra.html#AdicCompletion.eval%E2%82%90).

What's the convention? Personally I perfer `liftAlgHom`.

(By the way, isn't `RatFunc.liftAlgHom` a special case of `IsFractionRing.liftAlgHom`? The condition `nonZeroDivisors (Polynomial K) ≤ Submonoid.comap φ (nonZeroDivisors L)` of that result is very strange... isn't it `Injective φ`? Because `Polynomial K` and `L` are both domains...)

Reply:
I think we should globally adapt the `ₐ` convention as it is shorter and clear enough.

Reply:
> I think we should globally adapt the `ₐ` convention as it is shorter and clear enough.

This seems reasonable to me, but it probably belongs to another PR.